docs/source/features/sheet_management/schema.rst

```diff
@@ -4,3 +4,217 @@
 ########
  Schema
 ########
+
+Each sheet has a record type schema to describe the name and the type of each column.
+
+Graviti use `Portex`_ schema language to define the schema, please refer to its documentation for the grammar.
+
+.. _Portex: https://github.com/Project-OpenBytes/portex
+
+Graviti SDK supports the python interaction with the Portex schema.
+
+
+*****************
+ Primitive Types
+*****************
+
+Graviti SDK provides classes to initialize Portex primitive types.
+
+.. code:: python
+
+   import graviti.portex as pt
+   pt.boolean()
+   pt.binary()
+   pt.string()
+   pt.int32()
+   pt.int64()
+   pt.float32()
+   pt.float64()
+
+***************
+ Complex Types
+***************
+
+Graviti SDK provides classes to initialize Portex complex types.
+
+enum
+====
+
+An enum type can be created by giving the values.
+
+.. code:: python
+
+   import graviti.portex as pt
+   enum = pt.enum(values=["a", "b", "c"])
+   enum.values
+
+.. note::
+   The values of the enum type must be strings with the format ``[A-Za-z_][A-Za-z0-9_]*``.
+
+array
+=====
+
+The array type can be created by giving the item type and length.
```

Comment:
here, length is not required. Should we move it into another sentence. like:
The array type can be created by giving the item type. param length can be specified to fix the length of array.